lib/data/recruitment_banners.yml

```diff
@@ -8,18 +9,3 @@
   #     - /
   #     - /foreign-travel-advice
 
-banners:
```

Comment:
Just wanted to double check whether we'd want to retain the empty `banners` here? It looks like it might have been previously kept there when there were no banners in the file: https://github.com/alphagov/government-frontend/blob/0f3357ee06cf69df2de0366ef30fbe7495ad4db3/lib/data/recruitment_banners.yml#L11

Reply:
Oh I think I missed that, apologies! Will approve the PR now.